plotnine/tests/test_annotation_stripes.py

```diff
@@ -76,3 +76,13 @@ def test_annotation_stripes_fill_direction_extend():
          )
 
     assert p == 'annotation_stripes_fill_direction_extend'
+
+
+def test_annotation_stripes_single_stripe():
+    p = (ggplot(df.assign(x=10))
+         + annotation_stripes()
+         + geom_point(aes('factor(x)', 'y'))
+         + geom_vline(xintercept=[0.5, 1.5, 2.5, 3.5])
+         )
```

Comment:
Can you use different colours for the stripe. For a single stripe, the default colour (grey) looks so much like the plot background. Does `geom_vline` have any purpose here?, in other tests the lines mark were we expect the boundaries of the stripes to be.